packages/sdk-bridge/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Unreleased
 
+### 1.0.0-rc.13
+
```

Comment:
Prob add something about bumping the configuration package?

Reply:
We're not bumping the configuration package here, considering we will not be releasing `rc.19` to production just yet.

Reply:
bumped to rc-18